go.mod

```diff
@@ -32,6 +32,7 @@ require (
 )
 
 require (
+	github.com/aws/aws-sdk-go v1.44.162
```

Comment:
seems to me like we don't use it, and use `aws-sdk-go-v2` instead, couldn't find any reference for it

Reply:
True, removed with `go mod tidy`.
Might be nice to add to the CI.